code/modules/mining/lavaland/ash_tree.dm

```diff
@@ -0,0 +1,82 @@
+//Houses the ash tree, a lava land tree that has been burning for quite some time making a maple like sweetener.
+
+/obj/structure/flora/ashtree
+	gender = PLURAL //same as other tree
+	icon = 'icons/obj/lavaland/ash_tree.dmi'
+	icon_state = "ashtree"
+	var/sap_icon_state = "ashtree_maple" //Are icon when we are full of honey or other sap
+	var/tabbed_icon_state = "ashtree_maple" //What we look like when tapping
+	name = "ashed tree"
```

Comment:
We prefer to have override variables **above** variable defines, it makes the code a bit cleaner.